docs/development_checklist.md

```diff
@@ -12,6 +12,6 @@ Before you commit any changes, check your work against these requirements:
 - New codeowners are added to `CODEOWNERS` (if applicable), using `python3 -m script.hassfest`
 - The `.coveragerc` file is updated to exclude your platform if there are no tests available or your new code uses a third-party library for communication with the device, service, or sensor. `config_flow.py` can't be excluded as it must be fully tested (100% coverage).
 - The `.strict-typing` file is updated to include your code if it provides a fully type hinted source.
-- The code is formatted using Black, as per these [guidelines](https://developers.home-assistant.io/blog/2019/07/31/black/). This can be done running the command `black --fast homeassistant tests`.
+- The code is formatted using Ruff (`ruff format homeassistant tests`).
```

Comment:
This should be enough:

```suggestion
- The code is formatted using Ruff (`ruff format`).
```

Reply:
Just tested and looks like the current directory is the default argument for `ruff format`, so even the `.` isn't needed, just `ruff format`. Very concise, thanks. =)